src/utils/styles/ListPageCreateButton.scss

```diff
@@ -0,0 +1,3 @@
+.list-page-create-button-margin {
+  margin: 1.5rem 0 1rem 0;
```

Comment:
are we sure we need this specific css?
is there a problem with the sdk components? we should let the sdk team know.
we should use PF or SDK Components, not overwrite css for basic components like the create button.

Reply:
Yes there is a problem with sdk component `ListPageHeader`, but solving that problem would take some time.. I think we don't have much time for waiting for somebody else to solve it. And I am worried that the change was intentional, because the component is made for displaying header. And when you display some header, there is some string usually, and when there is some string and some button, you usually display the string in the left, and the button in the right.